CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
 # Changelog
 
-## v1.1.4 - 2022-06-16
+## v1.1.4 - TBD
+
+### Adds
+
+- Add format_nongpl extra to jsonschema install. This ensures draft7 format checkers validate format adherence as expected while also ensuring GPL-Licenced transitive dependencies are not installed.
 
 ### Changes
 
 - Update jsonschema schema version dependency so that only versions greater than 4.4 are supported.
 
+### Removes
+
+- Native support for `iri` and `iri-reference` format checkers. This was removed because the format is implemented per the `rfc3987` jsonschema transitive dependency which is licensed under GPL.
```

Comment:
Perhaps:

```suggestion
- Automatic support for `iri` and `iri-reference` format checkers. This was removed because these format checkers require the `rfc3987` library, which is licensed under GPL. If you require these checkers, you can manually install `rfc3987` or install this package as `jsonschema[rfc3987]`.
```